src/modules/musig/session_impl.h

```diff
@@ -226,42 +226,55 @@ static int secp256k1_xonly_ge_serialize(unsigned char *output32, secp256k1_ge *g
     return 1;
 }
 
+/* Write optional inputs into the hash */
+static void secp256k1_nonce_function_musig_helper(secp256k1_sha256 *sha, unsigned int n_bytes_len, const unsigned char *extra_in32) {
```

Comment:
Nit: I can see two possible points of confusion here:
- `extra_in32` could get confused with the `extra_in` auxiliary input
- `n_bytes_len` could get confused for the length of `extra_in32` because the `n, data` pattern is so common

How would you feel about these renames?
- `extra_in32` -> `data32`
- `n_bytes_len` -> `prefix_size`

Reply:
Oh yeah, done.